RELEASE-NOTES.md

```diff
@@ -3,8 +3,30 @@
 ### Version 4.0.0-activity-tracker-unauthorised-beta
```

Comment:
Where's the entry for `4.1.6`, which is the version mentioned in `package.json`?

Reply:
this is to sync the master->Release, I have updated the version to 4.1.6-sync-master now, as this is a PR requrested by Mo to resolve conflicts merge from master->Release, personally I just took what is in master as priority, and then take the latest changes from Release by compare both branches, if it's not accurate enough on merging stuff from Release branch, as I don't have an accurate answer on some specific conflicts from Release branch, and it needs to go back the specific developer to resolve the merge conflicts regarding Release changes.